### PR TITLE
fix bug what can't see post on map after create

### PR DESCRIPTION
**Pull request by yuda1124**
so you can see your post on map after creating

### CODE REVIEW

File: shared/components/create.js [hunk 1]
```diff
@@ -264,11 +264,11 @@ class Create extends Component {
     .then((response) => response.json())
```
Comment (ovekyc) [inline shared/components/create.js]: response =>

File: shared/components/create.js [hunk 2]
```diff
@@ -264,11 +264,11 @@ class Create extends Component {
...
     .then((json) => {
```
Comment (ovekyc) [inline shared/components/create.js]: json =>

File: shared/components/create.js [hunk 3]
```diff
@@ -264,11 +264,11 @@ class Create extends Component {
...
       this.props.setPostedKey(json.data.itemKey);
-      this.handleSceneTransition()
+      this.handleSceneTransition();
     })
     .catch((error) => {
```
Comment (ovekyc) [inline shared/components/create.js]: error =>

File: shared/components/create.js [hunk 4]
```diff
@@ -351,19 +351,19 @@ class Create extends Component {
       .then(() => this.handleSceneTransition())
       .catch((error) => {
```
Comment (ovekyc) [inline shared/components/create.js]: error =>

File: shared/components/create.js [hunk 5]
```diff
@@ -297,12 +297,12 @@ class Create extends Component {
           });
         });
       } catch (error) {
-        console.log(error);
+        console.log(error); // eslint-disable-line no-console
       }
     },
     (error) => {
```
Comment (ovekyc) [inline shared/components/create.js]: error =>

File: shared/components/create.js [hunk 6]
```diff
@@ -207,7 +207,7 @@ class Create extends Component {
         this.setState({img: file});
       })
       .catch((err) => {
```
Comment (ovekyc) [inline shared/components/create.js]: err => console.log(err.message, err.code));

File: shared/actions/authActions.js [hunk 7]
```diff
@@ -66,80 +57,39 @@ const setSecretToken = async (secret) => {
...
-    setRefreshToken(rjson.refreshToken);
-    setUserKey(rjson.userKey);
-    return null;
-  })
-  .catch((error) => {
-    console.log(error); // eslint-disable-line no-console
-  });
+export const signupFacebookUser = (FacebookToken) => {
+  return UserRESTManager.signupFacebook(FacebookToken)
+    .then((rjson) => {
```
Comment (ochanje210) [inline shared/actions/authActions.js]: rjson =>

File: shared/components/create.js [hunk 8]
```diff
@@ -243,11 +243,30 @@ class Create extends Component {
...
       });
     })
-    .then(this.props.requestAddItem)
-    .then(() => this.handleSceneTransition())
-    .catch((error) => {
+    .then(data => {
+      dataFlag = data;
+      return getAccessToken();
+    })
+    .then(accessToken => {
```
Comment (ochanje210) [inline shared/components/create.js]: why don't you use RESTAPI from utils

**Pull request merged**